src/gengowatcher/notifier.py

```
@@ -2,91 +2,112 @@
+        # 5. aplay (ALSA - absolute fallback)
+        players = [
+            (["pw-play", str(path)], "pw-play"),
+            (["paplay", str(path)], "paplay"),
+            (["canberra-gtk-play", "-f", str(path)], "canberra-gtk-play"),
+            (["mpv", "--no-video", "--no-terminal", str(path)], "mpv"),
+            (["aplay", "-q", str(path)], "aplay"),
+        ]
+
+        for cmd, name in players:
+            try:
+                subprocess.run(
```

Ruff (0.15.2)

[error] `subprocess` call: check for execution of untrusted input

(S603)